dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/redshift/AbstractAwsApiTask.java

```diff
@@ -33,50 +31,33 @@
 import java.io.Writer;
 import java.nio.charset.StandardCharsets;
 import java.util.Optional;
-import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 import org.apache.commons.csv.CSVFormat;
 import org.apache.commons.csv.CSVPrinter;
 
 /** Abstract class that provides methods for connecting with AWS API and writing results. */
 public abstract class AbstractAwsApiTask extends AbstractTask<Void> {
 
-  AWSCredentialsProvider credentialsProvider;
   Class<? extends Enum<?>> headerEnum;
-  Optional<AmazonRedshift> redshiftClient;
-  Optional<AmazonCloudWatch> cloudWatchClient;
+  AmazonRedshift redshiftClient;
+  AmazonCloudWatch cloudWatchClient;
```

Comment:
cloudWatchClient used only in RedshiftClusterUsageMetricsTask, so it should not be part of abstract class